spec/src/main/asciidoc/converters.asciidoc

```diff
@@ -36,16 +36,7 @@ The following `Converter` s are provided by Microprofile-Config by default:
 * `long` and `Long`
 * `float` and `Float` , a dot '.' is used to separate the fractional digits
 * `double` and `Double` , a dot '.' is used to separate the fractional digits
-* `Duration`
```

Comment:
I don't see why these are removed, but `Long`, and other `Number` impls remain.

Reply:
I left them as we need to keep the primitive. I can remove the boxed type. However, I think it is better to keep them because they are kind of special classes.